core/modules/qproc/geomAdapter.h

```diff
@@ -0,0 +1,105 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2015 AURA/LSST.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+#ifndef LSST_QSERV_QPROC_GEOMADAPTER_H
+#define LSST_QSERV_QPROC_GEOMADAPTER_H
+/**
+  * @file
+  *
+  * @brief Geometry adapter interface code
+  *
+  * @author Daniel L. Wang, SLAC
+  */
+
+// System headers
+#include <vector>
+
+// Other external headers
+#include "boost/make_shared.hpp"
+
+// LSST headers
+#include "sg/Box.h"
+#include "sg/Circle.h"
+#include "sg/Ellipse.h"
+#include "sg/ConvexPolygon.h"
+
+// Qserv headers
+#include "css/StripingParams.h"
+#include "qproc/QueryProcessingBug.h"
+
+
+namespace lsst {
+namespace qserv {
+namespace qproc {
+typedef double Coordinate;
```

Comment:
I dislike this typedef, because you are using it for things that I wouldn't classify as a coordinate in the traditional sense of the word (e.g. circle opening angle, ellipse semi-major/semi-minor axis, etc...). Can we just use double?